src/web.ts

```diff
@@ -4,10 +4,11 @@ import type { ScreenSizePlugin } from './definitions';
 
 export class ScreenSizeWeb extends WebPlugin implements ScreenSizePlugin {
 
-  async getDevicePPI(): Promise<{ density: string }> {
+  async getDevicePPI(): Promise<{ density: string, scaleFactor: string }> {
     console.log('Accessing the PPI is not supported within a browser (web)!');
     return {
-      density: '0.0'
+      density: '0.0',
```

Comment:
why is it a string not a number?

Reply:
The first implementation went on the assumption that values returned from a plugin call needed to be a string to be readable in web code. After testing numbers as return values, the function is now changed to return numbers instead of strings to simplify the calculation, in which they are used.